Scrabble.java

```diff
@@ -84,10 +103,25 @@ public static void playHand(String hand) {
 			// Reads the next "token" from the keyboard. A token is defined as a string of 
 			// non-whitespace characters. Whitespace is either space characters, or  
 			// end-of-line characters.
+			
 			String input = in.readString();
-			//// Replace the following break statement with code
-			//// that completes the hand playing loop
-			break;
+			if (input.equals(".")){
+				break;
+			   }
+
+			if(!MyString.subsetOf(input, hand)){
+				System.out.println( "Invalid word. Try again.");
+			}

```

Comment:
Use `continue`. 
Could be structured way better.. 
else {
   if () {
  
  }

use else if